app/models/cms/section.rb

```diff
@@ -73,7 +73,7 @@ def partial_paths
     end
 
     def find_or_create!(name, path, options = {})
-      system_name = name.downcase
+      system_name = name.parameterize
```

Comment:
Why are we changing from `downcase` to `parameterize`?
After this change is deployed, couldn't we end-up with something like
```
Section1 -> my section
Section2 -> my-section
```
because of this `find_or_create!`?

Reply:
`parameterize` is to generate a valid `system_name` for the new format enforcing. In your examples:

```
Section1 -> section1
Section2 -> section2
```